### ISSUE TITLE
"run" command error: Double quote to prevent globbing and word splitting [shellcheck]

### ISSUE THREAD

**Issue by altjx** [koalaman/shellcheck#2422]
I have been trying to understand what exactly this means:

```
2021-12-26 22:37:50 [INFO]   File:[/github/workspace/.github/workflows/linter.yml]
2021-12-26 22:37:50 [ERROR]   Found errors in [actionlint] linter!
2021-12-26 22:37:50 [ERROR]   Error code: 1. Command output:
------
.github/workflows/linter.yml:50:9: shellcheck reported issue in this script: SC2086:info:1:8: Double quote to prevent globbing and word splitting [shellcheck]
   |
50 |         run: "git -C ${GITHUB_WORKSPACE} diff --name-only origin/${DEFAULT_BRANCH}...${GITHUB_SHA} --diff-filter=d \"**.rb\" | xargs bin/bundle exec rubocop -c .github/linters/.rubocop.yml --fail-level E --display-only-fail-level-offenses"
```

It's telling me to use double quotes, but I am. Any chance this is a bug? Initially, it was complaining about not having double quotes, so I added it and now still seeing the same thing. Not quite sure what else there is to double quote in this scenario.

Pretty sure this is a small issue on my end but I just can't figure out what.

**Comment by TinCanTech** [comment]
> It's telling me to use double quotes

Try `shellcheck -S warning`

**Comment by altjx** [comment]
Hey @TinCanTech I'm using SuperLinter, which uses `actionlint` which pulls from `shellcheck`. Just realized this. I think I'll probably just have to revert to `shellcheck` itself rather than through SuperLinter.

**Comment by TinCanTech** [comment]
I answered because I can see the fix. I thought it would be simple to configure the linter..

**Comment by altjx** [comment]
I don't think there's many configurable options using SuperLinter. I'd have to use the Shellcheck linter specifically (outside of SuperLinter). Since that's the case, I'd consider this resolved.
